src/server/routes.js

```diff
@@ -58,18 +60,22 @@ app.post("/inventory/product", function (req, res) {
   const document = req.body;
 
   if (
-    !document.product_id ||
```

Comment:
**Annotation**: Bang operator returning a false positive; switched to using strict comparison.